validator/Makefile

```diff
@@ -87,7 +87,7 @@ $(BUILD_TARGETS): build-%:
 		--build-arg VERSION="$(VERSION)" \
 		--build-arg GIT_COMMIT="$(GIT_COMMIT)" \
 		--build-arg BUILDER_IMAGE="$(BUILDER_IMAGE)" \
-		--build-arg CUDA_SAMPLE_IMAGE=nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda$(CUDA_SAMPLES_VERSION)-$(*) \
```

Comment:
@cdesiniotis . If you see here, the cuda samples container image reference resolves to `nvcr.io/nvidia/k8s/cuda-sample:vectoradd-cuda11.7.1-ubi8`.  As we are no longer using the `ubi8` tag, I thought it would be preferable to use a cuda-sample image that has a tag without `ubi8`.  Also, this way, the cuda-sample image has nothing to do with the base distribution we use to build the gpu-operator image.

Reply:
Sure, I understand wanting to move to an image tag that is distro-agnostic. I am more concerned with the CUDA_SAMPLES_VERSION bump. As long as all active driver branches are compatible with 12.5.0, then I am fine with this change.

Reply:
Yes, that should be the case. Since the e2e tests cover both the 535 and 550 drivers, we can confirm that the new cuda samples image is working for the active driver branches  we support